index.html

```diff
@@ -1,82 +1,111 @@
 <!DOCTYPE html>
 <html lang="en-us">
 
+<!-- Head -->
 <head>
     <meta charset="UTF-8" />
+    <!-- Link rel to CSS stylesheet -->
     <link rel="stylesheet" href="./assets/css/style.css">
-    <title>website</title>
+    <!-- Website title -->
+    <title>Horiseon</title>
 </head>
 
+<!-- Body -->
 <body>
-    <div class="header">
+    <!-- Website header -->
+    <header>
         <h1>Hori<span class="seo">seo</span>n</h1>
-        <div>
+      
+        <!-- Navigation menu -->
+        <nav>
             <ul>
+                <!-- Navigation Link Search Engine Optimisation -->
                 <li>
                     <a href="#search-engine-optimization">Search Engine Optimization</a>
                 </li>
+                <!-- Navigation Link Online Reputation Mangement -->
                 <li>
                     <a href="#online-reputation-management">Online Reputation Management</a>
                 </li>
+                <!-- Navigation Link Social Media Marketing -->
                 <li>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+        </nav>
+
+    </header>>
```

Comment:
Looks like you have a typo here
```suggestion
    </header>
```
If you open the HTML file in the browser then you will see a `>` bracket somewhere after the header. This is a common typo and something to be aware of as I still make this mistake.